AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -90,7 +90,7 @@ - (void)ingestionDidReceiveFatalError:(__unused id<MSIngestionProtocol>)ingestio
 
 #pragma mark - Managing queue
 
-- (void)enqueueItem:(id<MSLog>)item {
+- (void)enqueueItem:(id<MSLog>)item critical:(BOOL)critical {
```

Comment:
Same here, should it be a bitmask instead?

Reply:
As discussed in #1189